bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -297,6 +297,7 @@ void PreProcessor::registerClientPreProcessRequest(ClientPreProcessReqMsgUniqueP
   {
     // Only one request is supported per client for now
     auto &clientEntry = ongoingRequests_[clientId];
+    LOG_INFO(GL, "bug(1)");
```

Comment:
Please remove all the logs you added for debugging.

Reply:
done